src/App.tsx

```diff
@@ -25,51 +25,51 @@ const Landing = lazy(() => import('pages/Landing'))
 const pageContainer = classnames(
   display('flex'),
   flexDirection('flex-col'),
+  overflow('overflow-hidden'),
   minHeight('min-h-screen')
 )
-const overflowEffects = overflow('overflow-hidden')
-const bodyContainer = classnames(
-  width('xs:w-full'),
-  maxWidth('max-w-fit', 'body:max-w-body'),
-  margin('mx-4', 'mb-auto', 'body:mx-auto')
-)
+const bodyContainer = (marginTop?: boolean) =>
+  classnames(
+    width('xs:w-full'),
+    maxWidth('max-w-fit', 'body:max-w-body'),
+    margin('mx-4', 'mb-auto', 'body:mx-auto', { 'mt-auto': marginTop })
+  )
 
 export default function () {
   const [location] = useLocation()
   const hasBlockchainList = location === '/' || location === '/cast'
+  const is404 = location === '/404'
 
   return (
     <Router>
       <div className={pageContainer}>
         <Navbar />
-        <div className={overflowEffects}>
-          <div className={bodyContainer}>
-            <Switch>
-              <Route path="/">
-                <LazyComponent lazyImported={<Landing />} />
-              </Route>
-              <Route path="/cast">
-                <LazyComponent lazyImported={<Cast />} />
-              </Route>
-              <Route path="/terms">
-                <Terms />
-              </Route>
-              <Route path="/privacy">
-                <Privacy />
-              </Route>
-              <Route path="/wallet">
-                <BurnerWallet />
-              </Route>
-              <Route path="/404">
-                <NotFound />
-              </Route>
-              <Route path="">
-                <Redirect to="/404" />
-              </Route>
-            </Switch>
-          </div>
-          <Footer />
+        <div className={bodyContainer(is404)}>
```

Comment:
removing `overflowEffects` with `overflow('overflow-hidden')` will break the `/wallet` page

Reply:
It shouldn't break it. Because I left `overflow('overflow-hidden')` in the block above